hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -574,8 +574,8 @@
                     <exclude>**/azurebfs/services/ITestReadBufferManager.java</exclude>
                     <exclude>**/azurebfs/ITestAbfsMsiTokenProvider.java</exclude>
                     <exclude>**/azurebfs/ITestAbfsRestOperationException.java</exclude>
+                    <exclude>**/azurebfs/ITestAzureBlobFileSystemAppend.java</exclude>
```

Comment:
Can we avoid this? Following things may help:
1. avoid making changes to original `Configuration` object which is being used to create new instance of FileSystem.

Reply:
Sequential may not be the correct way here. Wrote a test:
```
  @Test
  public void test() throws Exception {
    for(int i=0;i<10;i++) {
      System.out.println("iteration: " + i);
      testParallelWriteOutputStreamClose();
      getFileSystem().delete(new Path("secondarytestfile"), true);
    }
  }
```

ran on IDE:
```
java.lang.AssertionError: 
Expected :200
Actual   :400
<Click to see difference>


	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at org.apache.hadoop.fs.azurebfs.ITestAzureBlobFileSystemAppend.testParallelWriteOutputStreamClose(ITestAzureBlobFileSystemAppend.java:540)
```
Need more debugging.